beta-src/src/state/game/game-api-slice.ts

```diff
@@ -333,6 +409,101 @@ const gameApiSlice = createSlice({
             setCommand(state, command, "mapCommands", "all");
           }
         }
+      } else if (
+        clickObject === "territory" &&
+        phase === "Builds" &&
+        currentOrders
+      ) {
+        const territoryMeta = territoriesMeta[Territory[territoryName]];
+        if (territoryMeta) {
+          const {
+            coast: territoryCoast,
+            countryID,
+            id: webDipTerritoryID,
+            supply,
+            type: territoryType,
+          } = territoryMeta;
+
+          if (member.countryID.toString() !== countryID || !supply) {
+            return;
+          }
+
+          const existingBuildOrder = Object.entries(ordersMeta).find(
+            ([, { update }]) => {
+              if (update) {
+                return update.toTerrID === webDipTerritoryID;
+              }
+              return false;
```

Comment:
should be able to do a ternary here
`=> update ? update.toTerrID === webDipTerritoryID : false;`